packages/deployments/contracts/contracts/lib/Connext/ConnextLogic.sol

```diff
@@ -17,14 +18,16 @@ import {TypeCasts} from "../../nomad-core/contracts/XAppConnectionManager.sol";
 import {Home} from "../../nomad-core/contracts/Home.sol";
 
 import {ECDSAUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/cryptography/ECDSAUpgradeable.sol";
-import {SafeERC20Upgradeable, AddressUpgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import {SafeERC20Upgradeable, IERC20Upgradeable, AddressUpgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
 
 library ConnextLogic {
   // ============ Libraries ============
   using TypedMemView for bytes;
   using TypedMemView for bytes29;
   using ConnextMessage for bytes29;
 
+  // TODO Get Aave referral code for Connext
```

Comment:
can we turn this into an issue?

Reply:
Created the following issue https://github.com/connext/nxtp/issues/1084